lua/luasnip/extras/fmt.lua

```diff
@@ -97,7 +99,11 @@ local function interpolate(fmt, args, opts)
 		-- The nodes are modified in-place as part of constructing the snippet,
 		-- modifying one node twice will lead to UB.
 		if used_keys[key] then
-			table.insert(elements, copy3(args[key]))
+			if opts.dup_copy then
+				table.insert(elements, copy3(args[key]))
+			else
+				table.insert(elements, rp(args[key].pos))
```

Comment:
Oh, some nodes (think `functionNode`) don't have a jump-indx (/`pos`), they should still be copied (it's fine in their case)